assets/Makefile

```diff
@@ -0,0 +1,36 @@
+NODE_VERSION=$(shell python3 -c "import json; print(json.loads(open('assets/package.json').read())['engines']['node'])")
+
+.PHONY: env-assets test-assets clean-assets cleanenv-assets
+
+# Assert that we're using the proper node.js version
+assert-assets-node:
+ifeq ($(shell node --version), v$(NODE_VERSION))
+	@echo 'Using node $(NODE_VERSION)'
+else
+	@echo 'Please use node.js version $(NODE_VERSION).'
+	@echo 'Consider nvm to manage node versions: https://github.com/creationix/nvm'
+	@exit 1
+endif
+
+env-assets: assert-assets-node assets/node_modules
+	@echo 'Created assets environment!'
+
+assets/node_modules:
```

Comment:
Caveat, I'm not a Makefile expert. Should installing the npm dependencies by part of env-assets?

Reply:
Yep - here, `assets/node_modules` (the directory) is the build target. If the directory doesn't exist, its build target is called, which does the `npm install`.